docker-compose.yml

```diff
@@ -21,13 +21,17 @@ services:
       target: development
       args:
         DOCKER_USER: ${DOCKER_USER:-1000}
-    image: "funmooc:development"
-    env_file: "env.d/development"
+        SITE: ${ACTIVE_SITE:-funmooc}
+    image: "richie:development"
```

Comment:
```suggestion
    image: "${ACTIVE_SITE:-funmooc}:development"
```

Reply:
well I don't really care how this image is called and I prefer to not pollute my system with too many different image names. I'm ok to do that if you think it's important.

Reply:
And it will also complicate the CI configuration as I will need to variabilize all image names.

Reply:
Yes, this is one of my review command. It' not that complicated IMO.

This is a label, it won't change cache layers. I think it's more relevant to have a proper name for your images as `richie` could target the core project development or any derived site...